bindings/python/src/lib.rs

```diff
@@ -54,17 +65,22 @@ fn prepare<'a>(tensor_dict: HashMap<String, &'a PyDict>) -> PyResult<HashMap<Str
 fn serialize<'a, 'b>(
     py: Python<'b>,
     tensor_dict: HashMap<String, &'a PyDict>,
+    tensor_data: Option<HashMap<String, String>>,
```

Comment:
It's nothing to do with tensors here, so would just call it `metadata`.

Reply:
`Extra_data` ?

It's Metadata within Metadata at some point. 
(Here it's not confusing because metadata does not exist, but if we can have the name throughout the code base it's easier to read I think)

Reply:
I kept `metadata`. I'm being to nit-picky here (the field is name `__metadata__` which is makes sense. we have 2 kinds of metadata but it's not that horrible I think.